tests/cases/fourslash/externalModuleNameCompletions2.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='fourslash.ts'/>
+
+// @Filename: ambients.ts
+////declare module "a" {}
+////declare module "b" {}
+
+// @Filename: moduleC.ts
+////export var x = 0;
+
+// @Filename: global.ts
+////var global = 0;
+
+// @Filename: current.ts
+////declare module "d" {
+////    import * from "/*1*/";
+////}
+
+goTo.marker("1");
+verify.memberListContains("a");
+verify.memberListContains("b");

```

Comment:
well.. just cause this is ambient, so you want it to depend on other ambient things. more of limiting the clutter in .d.ts completion list.